0.22.0 release (#1051)
packit/packit: CHANGELOG.md

# 0.22.0

> - `packit init` introduces `--upstream-url` option. When specified, then
>   a source-git repository is created instead of just configuration file.

Reviewer: ```suggestion
- `packit init` introduces the `--upstream-url` option. When specified,
  `init` also sets up a source-git repository next to creating a configuration file.
```

> - Don't rewrite macros when setting release and version in spec file.

Reviewer 2: ```suggestion
- Don't rewrite RPM macros when setting release and version in a spec file.
```
